ajax/libs/wangEditor/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "wangEditor",
+  "filename": "js/wangEditor.min.js",
+  "version": "2.0.15.1",
+  "description": "wangEditor - 基于javascript和css开发的 Web富文本编辑器， 轻量、简洁、易用、开源免费",
+  "homepage": "http://wangeditor.github.io/",
```

Comment:
I'll suggest you consider to use `https` instead of `http` for __github.io__ domain.